sql/moz-fx-data-shared-prod/telemetry_derived/firefox_use_counters_v1/query.sql

```diff
@@ -0,0 +1,16623 @@
+-- Query for telemetry_derived.firefox_use_counters_v1
+            -- For more information on writing queries see:
+            -- https://docs.telemetry.mozilla.org/cookbooks/bigquery/querying.html
+---------SECTION 1 - FIREFOX DESKTOP ---------
+WITH firefox_desktop_use_counts_by_day_version_and_country_stg AS (
+  SELECT
+    DATE(a.submission_timestamp) AS received_date,
```

Comment:
In general, in majority of our queries we would call this field `submission_date`, maybe we should do the same to avoid each table using a different field name for this.

Reply:
good idea, will update this, thanks!